ang/civiawards-base/services/applicant-management-workflow.service.js

```diff
@@ -103,15 +148,15 @@
     }
 
     /**
-     * @param {object[]} workflows list of workflows
-     * @returns {object[]} formatted list of workflows
+     * @param {object[]} results results
+     * @returns {object[]} formatted results
      */
-    function getFormattedAwardDetailsData (workflows) {
-      var managerContacts = getAwardManagersForWorkflows(workflows);
+    function getFormattedAwardDetailsData (results) {
```

Comment:
there are a couple of things here:

* `getFormattedWorkflowsList` and `getWorkflowsList` should return an object instead of an array. `{get: {}, count: {}}` maybe? The reason is that it's difficult to know which index belongs to which result.
* `getFormattedAwardDetailsData` should only accept workflows as it was doing before, it should not know that the results is an array object containing workflows and counts. Since it won't do anything about the counts it should only accept workflows so we avoid confusing the purpose of the function.

I think `getFormattedWorkflowsList` and `getWorkflowsList` need to be refactored in a way similar to this:

```js
function getFormattedWorkflowsList (...) {
  // ...
  var apiCalls = {
    get: ['Award', 'get', paramsWithLimit],
    count: ['Award', 'getcount', params]
  };

  return civicaseCrmApi(apiCalls).then(function (data) {
    return {
      get: getFormattedAwardDetailsData(data.get.values),
      count: data.count
    };
  });
}
```